src/main/webapp/profile.html

```diff
@@ -39,36 +40,48 @@
       </nav>
       <br>
       <h1 class="text-center">Your Profile</h1>
-      <form>
-        <!-- TODO: fill in values based on user data from Datastore. -->
-        <div class="form-group row">
-          <label for="name" class="col-sm-2 col-form-label">Name</label>
-          <div class="col-sm-10">
-            <input type="text" readonly class="form-control-plaintext editable" id="user-company" value="Tess Ter">
-          </div>
-        </div>
-        <div class="form-group row">
-          <label for="userEmail" class="col-sm-2 col-form-label">Email</label>
-          <div class="col-sm-10">
+      <form class="needs-validation" novalidate action="/register" method="POST">
+        <div class="form-row">
+          <div class="col-md-12 mb-3">
+            <label for="user-email">Email</label>
+            <!-- TODO: replace with user's actual email -->
             <input type="text" readonly class="form-control-plaintext" id="user-email" value="tess@gmail.com">
           </div>
         </div>
-        <div class="form-group row">
-          <label for="user-company" class="col-sm-2 col-form-label">Company</label>
-          <div class="col-sm-10">
-            <input type="text" readonly class="form-control-plaintext editable" id="user-company" value="Google">
+        <div class="form-row">
+          <div class="col-md-6 mb-3">
+            <label for="first-name-field">First name</label>
+            <input type="text" class="form-control-plaintext editable" id="first-name-field" placeholder="First name" value="Tess" required>
+          </div>
+          <div class="col-md-6 mb-3">
+            <label for="first-name-field">Last name</label>
+            <input type="text" class="form-control-plaintext editable" id="last-name-field" placeholder="Last name" value="Ter" required>
           </div>
         </div>
-        <div class="form-group row">
-          <label for="user-job" class="col-sm-2 col-form-label">Job</label>
-          <div class="col-sm-10">
-            <input type="text" readonly class="form-control-plaintext editable" id="user-job" value="Software Engineer">
+        <div class="form-row">
+          <div class="col-md-6 mb-3">
+            <label for="company-field">Company</label>
+            <input type="text" class="form-control-plaintext editable" id="company-field" value="Google">
+          </div>
+          <div class="form-group col-md-6 mb-3">
+            <label for="job-field">Job</label>
```

Comment:
Why is this showing up as a dropdown even when it's set?

Reply:
Software Engineer is the first option

Reply:
When we are showing the user the current info they inputted, we shouldn't have the job entry look like it's still modifiable.

Reply:
Updated screenshots.